src/concept/answer/ConceptMap.jl

```diff
@@ -17,11 +17,18 @@ function Base.show(io::IO, cm::ConceptMap)
     num_entries = length(cm.data)
     print(io, "ConceptMap:\n")
     for (i, k) in enumerate(sort(collect(keys(cm.data))))
-        i != num_entries && print(io, "\n")
-        print(io, lpad("$i", 5), ". ", k, " => ", cm.data[k])
+        #TODO: decide which type of printing looks better. For me (Frank) it looks better
+        #with a line break after each line.
+        #i != num_entries && print(io, "\n")
+        print(io, lpad("$i", 5), ". ", k, " => ", cm.data[k], "\n")
```

Comment:
1. No need to interpolate: `lpad(i, 5)` should work.
2. You can just use `println` instead of adding `"\n"` at the end.

Reply:
1.  it's your code 😄 , I only commented out line 22
2. it's your code 😄